docs/schema/V1/swagger.verified.json

```diff
@@ -3986,17 +4037,27 @@
         "additionalProperties": false,
         "properties": {
           "consumerType": {
-            "$ref": "#/components/schemas/AttachmentUrlConsumerType_Values"
+            "description": "The type of consumer the URL is intended for.",
+            "oneOf": [
```

Comment:
Hm, weird .. here we have one going from single to oneOf 🤷🏼
Gonna have a look at #1183

Reply:
Seems like "description" is not allowed alongside "$ref" in 3.0, but it is allowed in 3.1:
https://github.com/OAI/OpenAPI-Specification/issues/1514#issuecomment-1386613645